examples/tabs/tabs-automatic.html

```diff
@@ -68,10 +121,10 @@ <h2 id="ex_label">Example</h2>
 
       <section>
         <h2>Accessibility Features</h2>
-        <p>
-          To demonstrate the effects of deleting a tab, the third tab, labeled <q>Joke</q>,
-          can be deleted when it has focus by pressing <kbd>Delete</kbd>.
-        </p>
+        <ul>
+          <li>The selected <code>tab</code> has a 2 pixel border when it is selected to make the selected tab discernible from the other tabs to people with visual impairments using operating system high contrast modes.</li>
```

Comment:
This sentence is a bit long and there's a bit of redundancy. I think we can remove some words to make it easier to read.
```suggestion
          <li>The selected <code>tab</code> has a 2 pixel border to make it discernible from other tabs, for people with visual impairments using operating system high contrast modes.</li>
```

Reply:
Fixed